gravitee-am-identityprovider/gravitee-am-identityprovider-api/src/main/java/io/gravitee/am/identityprovider/api/UserProvider.java

```diff
@@ -37,6 +37,8 @@ default Maybe<User> findByEmail(String email) {
 
     Single<User> update(String id, User updateUser);
 
+    Completable updateUsername(String oldUsername, String username);
```

Comment:
The first parameter is called `oldUsername` in the interface and `id` in all implementations
I think they should be in line

Reply:
This is an old revision. 
The last commit 

```
 default Single<User> updateUsername(User user, String username) {
        if (username == null || username.isEmpty()) {
            return Single.error(new IllegalArgumentException("Username required for UserProvider.updateUsername"));
        }
        ((DefaultUser) user).setUsername(username);
        return this.update(user.getId(), user);
    }
```